Core Updates

Dunrab opened the pull request:
# Description

I used an older version of core pre previous changes in order to make all the below changes easier to make.

A lot of fixes for Core:

- changed the SM back to a singulo, its not core with out a singulo
- added a justice wing to the station
- the old lawyer office was turned into a cafe for service workers to run
- added an entire new dorms area
- fixed cryosleep not having a station beacon
- added a mail room in logi for couriers
- added a proper teg setup so its not cramped in a corner of atmos along with a burn chamber
- fixed paramedlockers so they now contain a hard suit
- moved the kitchen to where the stage used to be
- replaced the old kitchen with a lounge area
- expadned evac to fit the evac shuttle and added an addtional docking spot to evac
- expanded salv dock so it dosent block logi shuttle from docking
- chem starts with plasma so they don't need to ask epi or logi on round start
- open the prober and server room so epi can access it with out the need for a mystagouge
- swapped mystagouges room with the old server room
- added holopads to each departement
- moved the shadowkin portal out of the maints bar to a separate room so the maints bar can be used/repaired
- prepared AI jobspawn and placed law boards (ai is not enabled in the map prototype)
- moved a weirdly placed musician spawn
- all of command now has access to the EVA room, not just the HOP
- gave atmos atmospheric alerts computers since they didn't have any
- added a psych office since the map lacked one
- added medical borg spawns medical 
- removed cameras from head's bed rooms, service bed rooms and old dorms

---

<details><summary><h1>Map Render</h1></summary>
<p>


<img width="5952" height="4160" alt="Core-0" src="https://github.com/user-attachments/assets/d4571b89-131e-45ba-9372-81a8e61cf86e" />




</p>
</details>

---

# Changelog
:cl:
- tweak: Many tweaks to Core to make the map more enjoyable & the singularity is back.

Dunrab committed:
core updates

see pr for full list of changes to much to list here

Dunrab committed:
add psych slot

add psych slot

Dunrab committed:
only need two paramed slots

changed paramed slots to two

Dunrab committed:
missed some decals

missed some dirt decals and made some areas look nicer

M3739 reviewed (dismissed):
All tests green, and everything seems to look good from a glance.

M3739 reviewed (commented):
Decided to look more into the map render. A more thorough review will come a bit later after the release. For now, there is no flooring below the new dorm entry doors.

<img width="202" height="158" alt="Image" src="https://github.com/user-attachments/assets/d20c857f-3d51-42ba-8061-b35d3a3f4887" />

<img width="263" height="243" alt="Image" src="https://github.com/user-attachments/assets/fea7432b-8450-4335-902c-e76e4e65cb75" />

Dunrab commented:
noticed one of the apcs i put in the court house is facing the direction need to fix that to

Mnemotechnician reviewed (approved):
I'll approve this despite the minor issues because the version of Core we have right now is an unholy abomination that really needs to be changed.

Dunrab committed:
small fixes

fixed tiles missing under doors
fixed maints not being accessible from the tool room because a table was blocking it
the apc i thought was facing the wrong way is facing the right way, the render paints it weird

Dunrab commented:
fixed tiles missing under doors
fixed maints not being accessible from the tool room because a table was blocking it
the apc i thought was facing the wrong way is facing the right way, the render paints it weird

M3739 commented:
In case of any future adjustments you wish to make, I won't merge this until the deployment PR is made later on, either tonight or tomorrow.

By then, I intend to merge this to prep for next release.

Dunrab committed:
removed cameras from bed rooms

removed cameras from head bed rooms, service bed rooms and cameras in old dorms.

Dunrab commented:
im glad someone caught it, wizden had cams in the bedrooms of all heads, dorms and some service bed rooms, removed them all.

M3739 commented:
Merging in 1 hour.
